assets/sass/components/analytics-4/_googlesitekit-analytics-setup-module.scss

```
@@ -55,6 +64,17 @@
 				letter-spacing: $ls-xs;
 				margin: 23px 0 0;
 			}
+
+			.googlesitekit-analytics-enable-enhanced-measurement {
+
+				.googlesitekit-module-settings-group__helper-text {
+					margin-left: 42px;
+				}
+			}
+
+			.googlesitekit-analytics-create-account-info {
+				margin-top: 1em;
```

We should add a bit of `padding-bottom` to better match the design:

```suggestion
				margin-top: 1em;
				padding-bottom: 6px;
```

---

Updated, thanks Tom!